### ISSUE TITLE
Extremely low performance on google cloud L4 Machines

### ISSUE THREAD

**Issue by aihkla** [google-ai-edge/mediapipe#5526]
### Have I written custom code (as opposed to using a stock example script provided in MediaPipe)

None

### OS Platform and Distribution

Google Cloud L4 Window 2022 server

### Task name (e.g. Image classification, Gesture recognition etc.)

Everything

### Programming Language and version (e.g. C++, Python, Java)

N/A

### Describe the actual behavior

Nothing, just opening the mediapipe offical studio on chrome

### Describe the expected behaviour

Run smooth like a regular nvidia PC like a 3080

### Standalone code/steps you may have used to try to get what you need

```shell
Basically I just turned on the google L4 cloud machine with nvidia driver 551.78.
With Intel Xeon CPU 2.20 X 16, 64GB ram and L4 GPU
```

**Comment by aihkla** [comment]
![image](https://github.com/google-ai-edge/mediapipe/assets/125827473/fa6105aa-c618-4295-a782-047ecd37bd02)
The performance is crazy, and i can confirm that the GPU performance runs perfectly fine on 3D application.

**Comment by kuaashish** [comment]
Hi @aihkla,

Our API layer currently supports GPU functionalities on macOS, regular Linux, and Windows for JavaScript, but not for Python on Windows. The scenario you are trying has not been tested, so sorry we can not provide information about its performance.

Thank you!!

**Comment by aihkla** [comment]
> Hi @aihkla,
> 
> Our API layer currently supports GPU functionalities on macOS, regular Linux, and Windows for JavaScript, but not for Python on Windows. The scenario you are trying has not been tested, so sorry we can not provide information about its performance.
> 
> Thank you!!

Hi, we are indeed using javascript on web in window PC using chrome. We are NOT using python on windows.
May I know if it is possible to allow mediapipe to run on CUDA core on the L4 with such setup?

Cuda with Mediapipe on JS  web <---is this possible? It is quite important for us, thank you.

**Comment by kuaashish** [comment]
Hi @aihkla,

We currently do not have documentation or test scenarios for this setup. If you are interested in support for this platform, please raise a feature request here. Based on future demand and discussions, we may implement this in the near future. We are sorry, but we can not help with this right now.

Thank you!!
